meta/main.yml

```diff
@@ -4,7 +4,7 @@ dependencies: []
 
 galaxy_info:
```

Comment:
Please add `role_name: repo_mirror` to ensure galaxy imports it correctly.

Reply:
Done